tests/integration/suite/test_notifier.py

```diff
@@ -0,0 +1,21 @@
+from .common import random_str
+
+
+def test_notifier_wechat_apiurl(admin_mc, remove_resource):
+    client = admin_mc.client
+    name = random_str()
+    defaultApiUrl = "https://qyapi.weixin.qq.com/cgi-bin/"
+    n = client.create_notifier(
+        clusterId="local",
+        name=name,
+        wechatConfig={"agent": "1111",
+                      "apiUrl": defaultApiUrl,
+                      "corp": "corp",
+                      "defaultRecipient": "2",
+                      "recipientType": "party",
+                      "secret": "231"})
+    remove_resource(n)
+    assert n is not None
+    assert (n["wechatConfig"]["apiUrl"]) == defaultApiUrl
```

Comment:
You shouldn't have to set this right? The default url is set when this is blank, and that functionality should be tested.